requirements-dev.txt

```diff
@@ -35,14 +35,10 @@ idna==2.10
     # via
     #   -c requirements.txt
     #   requests
-jinja2==3.0.3
+importlib-metadata==5.0.0
```

Comment:
why are dev dependencies being added when we remove a requiremnt?

Reply:
Not sure -- but they should be there either way. importlib-metadata is a package that gets pulled in on python installation versions less than 3.8. It comes in unpinned (as far as I can tell) [from the click setup.py](https://github.com/pallets/click/blob/8.0.x/setup.py#L7). I don't know why it wasn't there before, but it could be that the different release version requirement didn't exist when i last generated the requirement file, or having a newer version of python 3.7 in my venv caused it to resolve differently, click may have republished a version in the interim, etc.